### ISSUE TITLE
ListView deprecated in SignInScreen

### ISSUE THREAD

**Issue by kachnitel** [kachnitel/RideTime#53]
### Error: 
```
Warning: ListView is deprecated and will be removed in a future release. See https://fb.me/nolistview for more information
- node_modules/expo/build/environment/muteWarnings.fx.js:27:24 in error
- node_modules/react-native/node_modules/fbjs/lib/warning.js:30:18 in printWarning
- node_modules/react-native/node_modules/fbjs/lib/warning.js:51:23 in <unknown>
- node_modules/react-native/Libraries/Utilities/warnOnce.js:29:10 in warnOnce
- node_modules/react-native/Libraries/react-native/react-native-implementation.js:70:6 in ListView
- node_modules/react-native-modal-filter-picker/src/index.js:15:14 in ModalFilterPicker
- node_modules/react-proxy/modules/createClassProxy.js:94:29 in instantiate
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:7324:26 in constructClassInstance
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:11479:6 in updateClassComponent
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17276:21 in performUnitOfWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17316:41 in workLoop
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17417:15 in renderRoot
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18423:17 in performWorkOnRoot
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18324:24 in performWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18285:14 in performSyncWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18169:19 in requestWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17969:16 in scheduleWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:6934:17 in enqueueSetState
- node_modules/react/cjs/react.development.js:335:31 in setState
- node_modules/@react-navigation/native/src/createAppContainer.js:377:22 in dispatch
- node_modules/@react-navigation/core/lib/module/getChildNavigation.js:1:1529 in <unknown>
* screens/SignInScreen.js:88:37 in maybeSignUpOrError
* screens/SignInScreen.js:40:48 in <unknown>
- node_modules/promise/setimmediate/core.js:37:14 in tryCallOne
- node_modules/promise/setimmediate/core.js:123:25 in <unknown>
- ... 8 more stack frames from framework internals
```

**Comment by kachnitel** [comment]
See https://github.com/hiddentao/react-native-modal-filter-picker/pull/14

**Comment by kachnitel** [comment]
Replace modal-filter-picker with own modal that contains `LocationPicker` and adds selection to the result list?
Also filter out selected from the list (`options={this.state.locations.filter((location) => !this.state.picked.contains(location)}`

**Comment by kachnitel** [comment]
Implemented in d67be9ea1373e09a4da3879d5c99ed16cfa00769
...somewhat. Height is set manually and doesn't work automatically if width is set. That can step aside for a bit though!
